src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx

```diff
@@ -67,16 +69,26 @@ export function TableContentHeaderComponent(props: {
         </StyledTableCell>
     );
     props.headerValues.forEach((headerValue): void => {
-        let headerTitle = "";
+        let headerTitle: string | JSX.Element = "";
         if (props.isCol) {
-            headerTitle = t(
-                `FilterValues.${props.colAttribute}.${headerValue.replace(".", "")}`
-            );
+            const colKey = headerValue.replace(".", "");
+            if (props.colAttribute === "microorganism") {
+                const microorganismKey = colKey as MicroorganismLabelKey;
+                headerTitle = getMicroorganismLabelService(microorganismKey);
+            } else {
+                headerTitle = t(`FilterValues.${props.colAttribute}.${colKey}`);
+            }
             if (props.tableOption === "relative") {
-                headerTitle = `${headerTitle} ${t("Results.Unit")}`;
+                headerTitle = (

```

Comment:
Why you put this in a span?

Reply:
Because headerTitle coud be a string, or an JSX.Element (italic names).